metadata_backend/api/handlers/restapi.py

```diff
@@ -1,7 +1,7 @@
 """Handle HTTP methods for server."""
 import json
 from math import ceil
-from typing import AsyncIterator, Dict, Iterator, List, Optional, Tuple, Union
+from typing import Any, AsyncIterator, Dict, Iterator, List, Optional, Tuple
```

Comment:
So according to PEP 585, these would instead be
- dict
- list
- tuple
- collections.abc.AsyncIterator
- collections.abc.Iterator

However, `Any` and `Optional` would still be imported from `typing`